custom_components/tuya_local/devices/smartmi_e1_purifier.yaml

```diff
@@ -0,0 +1,61 @@
+name: Smartmi Air Purifier E1
+products:
+  - id: pwrmggdhwkz1lnvh
+    name: Smartmi Air Purifier E1
```

Comment:
Don't repeat the "Air purifier here", just the brand and model is enough (I haven't decided exactly how it will be used in future, but most likely displayed during device detection, and used for the model and manufacturer info for the device.